docs/source/guides/data.rst

```diff
@@ -0,0 +1,548 @@
+
+.. testsetup:: *
+
+    from pytorch_lightning.core.Lightning import LightningModule
+    from torch.utils.data import IterableDataSet
+    from pytorch_lightning.trainer.trainer.Trainer import Trainer
+
+.. _data:
+
+#############
+Managing Data
+#############
+
+Continue reading to learn about:
+
+* `Data Objects in Lightning <Data Objects in Lightning_>`_
```

Comment:
I think adding PyTorch first would be beneficial

Data Objects in PyTorch

Data Objects in Lightning

Reply:
Samples could be added too.

Reply:
I'm not sure i agree, because lightning supports passing dataloaders as well. Why would you separate?

Reply:
Good ! Resolving this one.